### ISSUE TITLE
Tag support for ingredients/catalysts

### ISSUE THREAD

**Issue by M4yd3** [oierbravo/create-mechanical-extruder#42]
I wanted to make a custom recipe with a tag instead of just an item, but it seems it's not supported. Any chance it will be, because I think this could add support for waterlogged blocks as well? Would be nice.

**Comment by TinyDeskEngineer06** [comment]
There isn't any tag for waterlogged blocks, it's just part of a block's blockstate. Adding tag support wouldn't allow support for waterlogged blocks.

**Comment by M4yd3** [comment]
Fair enough, that's why I wrote it **could** add support, not would. It would still be nice to add stuff to a taglist and then make one recipe instead of dozens, at least for the player experience.
Also adding separate support for waterlogged blocks wouldn't hurt, since vanilla cobblestone gens work with them just fine and you get used to it.
